### PR TITLE
Docs: Add instruction for creating Hello World's style.scss file

### PR DESCRIPTION
**Pull request by Tiendq**
I found that there is no instruction for styling a component in Hello World example, it makes new users confused especially who new to React (I saw people asking about it).

I could add a small section here about creating a `style.scss` file for the component, and importing it in to the main style file with example code. It's simple and mentioned in the Sass/CSS coding guidelines but it's naturally if we could say about it here for newcomers.

### CODE REVIEW

**Comment by hoverduck** [comment]
@Tiendq Feel free to add any documentation you think would be helpful.  Thanks!  And let us know if you have any questions.

It's probably also worth linking to the existing CSS documentation:

[Stylesheet README](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/README.md)
[CSS Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md)

**Comment by Tiendq** [comment]
@hoverduck Thank you, I'll let you know when it's done and needs a review.

**Comment by Tiendq** [comment]
@hoverduck I added the style section, please review. Thank you.

File: docs/guide/hello-world.md [hunk 1]
```diff
@@ -168,7 +169,40 @@ export default class HelloWorld extends React.Component {
...
+
+We'll create an empty `style.scss` file with the following command:
+
+```
+touch client/my-sites/hello-world/style.scss
+```
+
+Then add some styles for our `HelloWorld` component:
+
+```
```
Comment (spen) [inline docs/guide/hello-world.md]: We could add syntax highlighting to the code blocks here to make the code prettier on both GitHub and DevDocs 💅
Here's an example achieved by changing the opening ` ``` ` to ` ```css `:  

```css
  .hello-world__title {
    color: #37a000;
    font-size: 3rem;
 }
```
I'll drop line comments on the other blocks - if you don't mind adding those syntax tags that would be awesome 🙂

File: docs/guide/hello-world.md [hunk 2]
```diff
@@ -168,7 +169,40 @@ export default class HelloWorld extends React.Component {
...
+.hello-world__title {
+  color: #37a000;
+  font-size: 3rem;
+}
+```
+
+Let's update `h1` tag from the component we wrote above to include our title style:
+
+```
```
Comment (spen) [inline docs/guide/hello-world.md]: ` ```jsx ` would be ideal here :)

File: docs/guide/hello-world.md [hunk 3]
```diff
@@ -168,7 +169,40 @@ export default class HelloWorld extends React.Component {
...
+<h1 className="hello-world__title">Hello, World!</h1>
+```
+
+We need to do one more step to include the component's style file in the main application style file. It's done by importing `style.scss` in `assets/stylesheets/_components.scss`, add following line at the end of `_components.scss`:
+
+```
```
Comment (spen) [inline docs/guide/hello-world.md]: ` ```css ` again here should highlight the import syntax I think!

File: docs/guide/hello-world.md [hunk 4]
```diff
@@ -168,7 +169,40 @@ export default class HelloWorld extends React.Component {
...
+@import 'my-sites/hello-world/style';
+```
+
+That's it. Please check out the [CSS/Sass Coding Guidelines](../coding-guidelines/css.md) to learn more about working with stylesheet in the project.
```
Comment (spen) [inline docs/guide/hello-world.md]: Minor nit: `stylesheet` should be plural here.

**Comment by Tiendq** [comment]
Hi @spen, thanks for your view, it definitely makes code blocks nicer :). I didn't know `jsx` tag before, it is even better than `javascript` for JSX blocks.

**Comment by spen** [comment]
Awesome! Nice work picking up on the other blocks too - thanks @Tiendq :)

File: docs/guide/hello-world.md [hunk 5]
```diff
@@ -168,7 +169,48 @@ export default class HelloWorld extends React.Component {
...
+
+We'll create an empty `style.scss` file with the following command:
+
+```
+touch client/my-sites/hello-world/style.scss
+```
+
+Then add some styles for our `HelloWorld` component:
+
+```css
```
Comment (gwwar) [inline docs/guide/hello-world.md]: I believe `sass` is supported as well

File: docs/guide/hello-world.md [hunk 6]
```diff
@@ -168,7 +169,48 @@ export default class HelloWorld extends React.Component {
...
+}
+```
+
+Let's update `h1` tag from the component we wrote above to include our title style:
+
+```jsx
+export default class HelloWorld extends React.Component {
+	render() {
+		return (
+			<Main>
```
Comment (gwwar) [inline docs/guide/hello-world.md]: Let's also update this to 
```
<Main className="hello-world" >
```

**Comment by gwwar** [comment]
Thanks @Tiendq and @spen. I left a few minor notes, but this looks like a great improvement to the documentation.

Let me know when you're ready, and I can help land this.

**Comment by Tiendq** [comment]
Thanks @gwwar, I fixed it. I also found `import` block is a bit nicer with `scss` than `sass` then I used `scss` tag :)

**Comment by gwwar** [comment]
Thanks @Tiendq and @spen!

**Pull request merged**